.github/workflows/cd.yml

```diff
@@ -0,0 +1,121 @@
+name: 'Spendr Continuous Deployment'
+
+on:
+  push:
+    branches: ['ci-cd-configurations']
+  pull_request:
+    branches: ['ci-cd-configurations']
+
+jobs:
+  build-server:
+    name: 'spendr server continuous deployment build'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '19'
+
+    - name: Install Dependencies
+      run: cd server && npm install
+
+    - name: Build Server
+      run: cd server && npm run build
+
+  build-client:
+    name: 'spendr client continuous deployment build'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '19'
+
+    - name: Install Dependencies
+      run: cd client && npm install
+
+    - name: Build Client
+      run: cd client && npm run build
+
+    - name: Upload production ready build files
+      uses: actions/upload-artifact@v3
+      with:
+        name: spendr-client-static
+        path: ./client/build
+
+  publish-to-docker-server:
+    name: 'spendr server continuous deployment - publish'
+    needs: build-server
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/ci-cd-configurations'
```

Comment:
I think (?) we need to change this to main

Same thing for the one in client

Reply:
Im going to add this to the sister PR which just changes things to main from the tester branch